src/corehost/cli/args.cpp

```diff
@@ -96,5 +139,7 @@ bool parse_arguments(
     pal::getenv(_X("DOTNET_HOSTING_OPTIMIZATION_CACHE"), &args.dotnet_packages_cache);
```

Comment:
@muratg https://github.com/dotnet/core-proposals/blob/master/DotNetCore-SharedPackageStore.md This should give an idea of how to author shared packages with runtime support. We will even provide you with `dotnet cache` command to crossgen and build such a store. Cc @ramarag